isce2_topsapp/delivery_prep.py

```diff
@@ -131,8 +131,8 @@ def format_metadata(nc_path: Path,
     metadata.update({"ogr_bbox": ogr_bbox,
                      "reference_scenes": all_metadata['reference_scenes'],
                      "secondary_scenes": all_metadata['secondary_scenes'],
-                     "sensing_start": f'{ref_props["startTime"]}Z',
-                     "sensing_stop": f'{ref_props["stopTime"]}Z',
+                     "sensing_start": f'{ref_props["startTime"]}',
+                     "sensing_stop": f'{ref_props["stopTime"]}',
```

Comment:
This may no longer require an f-string, it might just be `"sensing_start": ref_props["startTime"],` (assuming that attribute is already a string and not a datetime object)